src/main/scala/mesosphere/mesos/Availability.scala

```diff
@@ -0,0 +1,25 @@
+package mesosphere.mesos
+
+import java.util.concurrent.TimeUnit
+
+import org.apache.mesos.Protos.Offer
+
+object Availability {
+
+  def offerAvailable(offer: Offer, drainingTime: Long): Boolean = {
```

Comment:
That's correct but later in the coude all this values are transformed to duration
```scala
  // Time to wait before trying to reconcile app tasks after driver starts
  val reconciliationInitialDelay =
    Duration(config.reconciliationInitialDelay(), MILLISECONDS)

  // Interval between task reconciliation operations
  val reconciliationInterval =
    Duration(config.reconciliationInterval(), MILLISECONDS)

  // Time to wait before trying to scale apps after driver starts
  val scaleAppsInitialDelay =
    Duration(config.scaleAppsInitialDelay(), MILLISECONDS)

  // Interval between attempts to scale apps
  val scaleAppsInterval =
    Duration(config.scaleAppsInterval(), MILLISECONDS)
```